src/frida/tracer.py

```diff
@@ -272,6 +288,10 @@ def _create_resolver_script(self):
             mappings[module.path] = index;
             return mappings;
         }, {});
+        cachedModules._idByName = cachedModules.reduce(function (mappings, module, index) {
+            mappings[module.name] = index;
```

Comment:
Let's move this assignment into the iteration right above so we only have to traverse `cachedModules` once.